### PR TITLE
AppVeyor: build on Windows with MSVC

### PR DESCRIPTION
**Pull request by pH5**
I have added a local Meson wrap for the OpenHMD hidapi fork and somehow made it build on AppVeyor. This does not include the OpenGL example yet.

### CODE REVIEW

**Comment by pH5** [comment]
Includes the fixes from #147.

File: examples/simple/simple.c [hunk 1]
```diff
@@ -11,7 +11,17 @@
...
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#define WIN32_EXTRA_LEAN
+#include <windows.h>
+static void ohmd_sleep(double seconds)
+{
+	Sleep((DWORD)(seconds * 1000));
+}
+#else
 void ohmd_sleep(double);
```
Comment (Wallbraker) [inline examples/simple/simple.c]: I feel like the right fix here is just export a ohmd_sleep function in the core API other then this hack. We should inhibit the amount of exported symbols on Linux as well.

Comment (pH5) [reply examples/simple/simple.c]: Both done.

**Comment by pH5** [comment]
> Is there away we can avoid committing a `tar.gz` into the repo? Is this something that can be solved inside of the Appveyor file?

We could commit the contained meson.build into our hidapi repository.
Or Meson could download the wrap patch from somewhere else, for example a binary attached to a hidapi release on our hidapi fork.

**Comment by TheOnlyJoey** [comment]
hmm since this is a temporary fix until libconvergence gets implemented (which probably will just use cmake and mason, like OpenHMD will after 0.3), i have no strong opinions about this.

**Comment by pH5** [comment]
Adapted to build against a hidapi fork that includes https://github.com/OpenHMD/hidapi/pull/2.

**Comment by pH5** [comment]
Updated to build hidapi from [OpenHMD/hidapi](https://github.com/OpenHMD/hidapi).

**Comment by pH5** [comment]
Any further comments?

**Pull request merged**